Gemfile

```diff
@@ -6,7 +6,7 @@ gem 'test-unit', '~> 3.2'
 
 gem 'bundler'
 
-gem "rails", "~> 6.0.4"
+gem "rails", "~> 6.0.4.8"
```

Comment:
Let's keep this 6.0.4, so Dependabot can work once https://github.com/dependabot/dependabot-core/issues/4867 is fixed.